aptos-move/framework/src/docgen.rs

```diff
@@ -15,8 +15,8 @@ pub struct DocgenOptions {
     #[clap(long)]
     pub include_impl: bool,
 
-    /// Whether to include specifications in the generated documentation. Defaults to true.
-    #[clap(long, global = true)]
+    /// Whether to include specifications in the generated documentation. Defaults to false.
+    #[clap(long)]
     pub include_specs: bool,
```

Comment:
The other option is we can make it exclude_specs.  This still lets it default to true, but allows for negation.

Reply:
It is probably fine.